build/pause/Makefile

```diff
@@ -132,9 +132,13 @@ container: .container-${OS}-$(ARCH)
 		-t $(IMAGE):$(TAG)-${OS}-$(ARCH) --build-arg BASE=${BASE} --build-arg ARCH=$(ARCH) .
 	touch $@
 
+.container-windows-$(ARCH): SHELL:=/bin/bash
 .container-windows-$(ARCH): $(foreach binary, ${BIN}, bin/${binary}-${OS}-${ARCH})
+	declare -A tagToKernelVersionMap=( ['1809']='17763' ['ltsc2022']='20348' ); \
+	kernel_version=$${tagToKernelVersionMap[$$OSVERSION]}; \
+	kernel_version_image_name=$$(./make-pause-windows.sh ${BASE.windows} $$kernel_version); \
 	docker buildx build --provenance=false --sbom=false --pull --output=type=${OUTPUT_TYPE} --platform ${OS}/$(ARCH) \
-		-t $(IMAGE):$(TAG)-${OS}-$(ARCH)-${OSVERSION} --build-arg BASE=${BASE}-windows-${OSVERSION}-${ARCH} --build-arg ARCH=$(ARCH) -f Dockerfile_windows .
+    		-t $(IMAGE):$(TAG)-${OS}-$(ARCH)-${OSVERSION} --build-arg BASE=$${kernel_version_image_name} --build-arg ARCH=$(ARCH) -f Dockerfile_windows .
```

Comment:
we shouldn't use names here, we can use the digest for the specific image, so for example mcr.microsoft.com/oss/kubernetes/windows-pause-image-base:v0.4.1@sha256:37cc10768383b55611d724a05eb18564cb5184c89b0c2faa7d4eff63475092df means that `sha256:37cc10768383b55611d724a05eb18564cb5184c89b0c2faa7d4eff63475092df` hashes to the manifest that contains the two manifests that themselves have digests `sha256:d4f254bb2a4e32cc5455e2bbc6c41f38c29ebe8ca18c6e6a19e816d5d711a164` and `sha256:8f49c039657e67cb54861d82acdb907104e268009f1bea4f75b3dec6c6b3d52d`

we can refer to them directly by only substituting the digest for the os.version specific digest